### ISSUE TITLE
HLS withCredentials fails

### ISSUE THREAD

**Issue by seesoe** [jwplayer/jwplayer#1928]
### Expected Behavior

Play HLS videos with ```withCredentials: true```

### Actual Behavior

JS error
```
Uncaught TypeError: Cannot read property 'extend' of undefined
```
```js
            return (e || t) && (r = _.extend({}, r, i(e, t))),
```

### Steps to reproduce

1. try playing a HLS resource with config set `withCredentials: true`

### Environment

- iMac, Chrome, Latest OS and browser.
- JW Player 7.10.1

**Comment by dannyfinks** [comment]
Hi @seesoe do you have a test page to show us? 

Are you setting withCredentials: true within the playlist block or globally in the configuration?

Also did this work in the previous version, 7.9.3?

**Comment by seesoe** [comment]
@dannyfinks , no this is on a local dev. I'm setting the flag on player setup and resource load. I get the error on page load with the initial file loaded on setup and when loading a new file into the player. I'll have to download 7.9.3 and test it. I'm currently using 7.9.0 with override hack.

**Comment by dannyfinks** [comment]
If you could provide us some type of sample it would go a long way to helping us resolve this. We were also aware of a similar issue where a cookie is not being set properly when using withCredentials which we are investigating for 7.10.2. I will let you know when we have 7.10.2 up on our beta channel to see if that resolved your issue.

**Comment by johnBartos** [comment]
@seesoe Thanks for the report, I found the bug. When upgrading our linter I accidentally deleted `_` from the module responsible for this behavior; the reason it didn't fail in our tests is because we include `_` as a global for our test pages 😅. This will be fixed in 7.10.2.
